node/src/GlideClusterClient.ts

```diff
@@ -638,6 +639,37 @@ export class GlideClusterClient extends BaseClient {
         return this.createWritePromise(createTime(), toProtobufRoute(route));
     }
 
+    /**
+     * Copies the value stored at the `source` to the `destination` key. When `replace` is `true`,
+     * removes the `destination` key first if it already exists, otherwise performs no action.
+     *
+     * See https://valkey.io/commands/copy/ for more details.
+     *
+     * @remarks When in cluster mode, `source` and `destination` must map to the same hash slot.
+     * @param source - The key to the source value.
+     * @param destination - The key where the value should be copied to.
+     * @param replace - (Optional) If `true`, the `destination` key should be removed before copying the
+     *     value to it. If not provided, no action will be performed if the key already exists.
+     * @returns `true` if `source` was copied, `false` if the `source` was not copied.
+     *
+     * since Valkey version 6.2.0.
+     *
+     * @example
+     * ```typescript
+     * const result = await client.copy("set1", "set2", { replace: true });
+     * console.log(result); // Output: true - "set1" was copied to "set2".
+     * ```
+     */
+    public async copy(
+        source: string,
+        destination: string,
+        options?: { replace?: boolean },
```

Comment:
Maybe you can simplify this for user, but it will be a bit incosistent with API in standalone client
```suggestion
        replace?: boolean,
```